PollBuddy-Server/frontend/src/components/Dropdown/Dropdown.js

```diff
@@ -51,7 +51,7 @@ function useOutsideAlerter(ref, menuProps) {
           console.log(response);
           if(response.result === "success") {
             //Logout has succeeded, Clear frontend user data
-            localStorage.setItem("loggedIn", false);
+            localStorage.setItem("loggedIn", "false");
```

Comment:
I'm also curious as to why you changed this to a string.

Reply:
This is how it is actually stored, it was not working before I changed it to a string.